base/task/sequence_manager/work_queue_unittest.cc

```diff
@@ -95,7 +95,7 @@ TEST_F(WorkQueueTest, Empty) {
 TEST_F(WorkQueueTest, Empty_IgnoresFences) {
   work_queue_->Push(FakeTaskWithEnqueueOrder(1));
   work_queue_->InsertFence(EnqueueOrder::blocking_fence());
-  EXPECT_FALSE(work_queue_->Empty());
```

Comment:
Good for testing, but lets back this out before landing

Reply:
Done